### ISSUE TITLE
Hardcoded routes for eoscpp

### ISSUE THREAD

**Issue by kesar** [ghoti143/eosio-builds#1]
After install EOS.IO-DAWN-2018-01-31-ALPHA-Linux.deb I tried to compile a smart contract with eoscpp but routes doesn't exist in my ubuntu version (  /home/ubuntu ) (I dont have ubuntu user)

/home/ubuntu/opt/wasm/bin/clang -emit-llvm -O3 --std=c++14 --target=wasm32 -ffreestanding -nostdlib -fno-threadsafe-statics -fno-rtti -fno-exceptions -I /home/ubuntu/eos/build/install/include -I . -c twitbot.cpp -o /tmp/tmp.Qot90fxOIH/built/twitbot.cpp
/usr/bin/eoscpp: line 36: /home/ubuntu/opt/wasm/bin/clang: No such file or directory

**Comment by ghoti143** [comment]
Yes -- good catch.  I've only ever used these builds to run eosiod -- but I should see if I can get all the executables to work.

This repo might start to become unneccesary, as it looks like the main EOS repo is starting to publish packaged builds:  https://github.com/EOSIO/eos/releases

Thanks!
